sqliterepo/sqlite_utils.c

```diff
@@ -500,3 +502,69 @@ version_extract_from_admin_tree(GTree *t)
 	return v;
 }
 
+struct container_properties_s
+{
+	GTree *system;
+	GTree *user;
+};
```

Comment:
"Container" is a meta2 thing. Container-related code should not be implemented in sqliterepo.